server/auth/oidc.js

```diff
@@ -28,8 +28,9 @@ const getStrategy = async (
     callbackUriBase
   );
   const params = {
-    scope: 'openid profile email',
+    scope: 'openid profile email read:default write:test0',
```

Comment:
we need to claim every role in the scope? is there a way to wildcard this?

Reply:
removed this commit from this PR. Wasn't meant to be here